src/components/Result/index.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import "./style.css";
+
+export default function renderResult({ result }) {
+  const renderList = () => {
+    // console.log(Object.values(result.repos));
+    const repoObjectArray = [];
+    for (const property in result.repos) {
+      const entry = Object.values(result.repos)[property];
+      repoObjectArray.push({
+        name: entry.name,
+        forks: entry.forks,
+        url: entry.html_url
+      });
+    }
+
+    return (
+      <div id="table">
+        <table>
+          <thead>
+            <tr>
+              <th>Repository Name</th>
+              <th>Number of Forks</th>
+            </tr>
+          </thead>
+          <tbody>
```

Comment:
what are the `thead` and `tbody` tags?

Reply:
table head and table body, useful as React doesn't like tr as a direct child of table